sway/tree/container.c

```diff
@@ -573,16 +573,32 @@ void container_update_title_textures(struct sway_container *container) {
 }
 
 void container_calculate_title_height(struct sway_container *container) {
+	sway_log(SWAY_INFO, "===> container_calculate_title_height");
+
+        int height;
+        int baseline;
+
 	if (!container->formatted_title) {
+		sway_log(SWAY_INFO, "===> container->formatted_title");
 		container->title_height = 0;
 		return;
 	}
-	cairo_t *cairo = cairo_create(NULL);
-	int height;
-	int baseline;
-	get_text_size(cairo, config->font, NULL, &height, &baseline, 1,
-			config->pango_markup, "%s", container->formatted_title);
-	cairo_destroy(cairo);
+	if (config->titlebar_v_height >= 0) {
+		sway_log(SWAY_INFO, "===> config->titlebar_v_height >= 0");
+
+		height   = config->titlebar_v_height;
+		baseline = round(height * 0.77);
```

Comment:
I'm not sure if I like this magic number, but I don't have any better ideas either. Any other reviewers have opinions about this?

Reply:
Since the value is not used elsewhere, the baseline value could also be a configuration directive.